fusor-ember-cli/app/templates/components/content-mirror-f.hbs

```diff
@@ -0,0 +1,14 @@
+{{#text-f label="Content Mirror URL" value=cdnUrl disabled=false}}
+  <div style="margin-top: 5px; margin-left: 3px">
+    {{#if isVerifyingContentMirror}}
+      <span>Verifying mirror...</span>
+      <span class="spinner spinner-md spinner-inline" style="margin-left: 5px; width: 15px; height: 15px"></span>
```

Comment:
Nitpicky, but while you're in there, you could probably do 

if isVerifyingContentMirror
else if isContentMirrorValid
else

instead of a nested if else.